src/organisms/forms/index.jsx

```diff
@@ -2,32 +2,83 @@ import React from "react";
 import { useForm, Controller } from "react-hook-form";
 import { TextField } from "@mui/material";
 import MenuItem from "@mui/material/MenuItem";
-import Select from "@mui/material/Select";
 import Button from "../../atoms/button";
 import useStore from "../../ions/hooks/storeFormData";
-import FormControl from "@mui/material/FormControl";
-import InputLabel from "@mui/material/InputLabel";
+import InputAdornment from "@mui/material/InputAdornment";
 
 const PlantForm = () => {
 	const plantCards = useStore(state => state.plantCards);
 	const setPlantCards = useStore(state => state.setPlantCard);
 
-	const [type, setType] = React.useState("");
+	const plantSpaces = [
+		{
+			value: "garden",
+			label: "garden",
+		},
+		{
+			value: "piece of land",
+			label: "piece of land",
+		},
+		{
+			value: "terrace",
+			label: "terrace",
+		},
+		{
+			value: "balcony",
+			label: "balcony",
+		},
+		{
+			value: "other",
+			label: "other",
+		},
+	];
+
+	const soilQualities = [
+		{
+			value: "clay",
+			label: "clay",
+		},
+		{
+			value: "sandy",
+			label: "sandy",
+		},
+		{
+			value: "loamy",
+			label: "loamy",
+		},
+		{
+			value: "chalky",
+			label: "chalky",
+		},
+		{
+			value: "silty",
+			label: "silty",
+		},
+
+		{
+			value: "peaty",
+			label: "peaty",
+		},
+	];
+
+	const [spaceType, setSpaceType] = React.useState("");
```

Comment:
If you import useState on top, like this:
```
import React, { useState } from "react";
```
you could use it as follows
```suggestion
	const [spaceType, setSpaceType] = useState("");
```